doc/classes/Animation.xml

```diff
@@ -38,6 +38,7 @@
 			<argument index="1" name="to_animation" type="Animation">
 			</argument>
 			<description>
+				Turns [code]to_animation[/code] into a copy of the animation at index [code]track[/code].
```

Comment:
Does it "turn" an existing animation to a copy of the given one, or *create* a new animation that's a copy?

Reply:
Oh woops, I wasn't looking at the code properly. Mistook one var for another. You're right.